tests/TestHarness/Cluster.py

```diff
@@ -1247,14 +1169,14 @@ def createSystemAccount(accountName):
             return None
 
         Node.validateTransaction(trans[1])
-        Utils.Print("Wait for issue action transaction to become finalized.")
+        Utils.Print("Wait for issue action transaction to appear in a block.")
         transId=Node.getTransId(trans[1])
-        # biosNode.waitForTransactionInBlock(transId)
+        biosNode.waitForTransactionInBlock(transId)
         # guesstimating block finalization timeout. Two production rounds of 12 blocks per node, plus 60 seconds buffer
-        timeout = .5 * 12 * 2 * len(producerKeys) + 60
-        if not biosNode.waitForTransFinalization(transId, timeout=timeout):
-            Utils.Print("ERROR: Failed to validate transaction %s got rolled into a finalized block on server port %d." % (transId, biosNode.port))
-            return None
+        #timeout = .5 * 12 * 2 * len(producerKeys) + 60
+        #if not biosNode.waitForTransFinalization(transId, timeout=timeout):
+        #    Utils.Print("ERROR: Failed to validate transaction %s got rolled into a finalized block on server port %d." % (transId, biosNode.port))
+        #    return None
```

Comment:
If this code has a future, let's let's document why it's disabled and the circumstances for us to re-enabling. If not, please remove.

Reply:
I thought it might need to be brought back when I was doing my commented code pass, but it looks like it's not required.  Removed.